### DISCUSSION TITLE
how to check if the png image is alpha chennel?

### DISCUSSION THREAD

**Discussion by lianxmfor** [ImageMagick/ImageMagick#5596] [Help] [answered]
I want to use the identify tool to check if the local png image is alpha channel, and I saw in one of the [StackOverflow answers](https://stackoverflow.com/a/2581516/16428442) that I can run the following command:

```shell
$ identify -format '%[channels]' foo.png 
srgba
```

The result of the command is srgba, I'm not quite sure what this means, is it srgb format + alpha channels?

**Comment by snibgo** [comment, marked as answer]
Yes, "srgba" means the image is encoded as sRGB, and it has an alpha channel.
